app/controllers/publish/providers/accredited_providers_controller.rb

```diff
@@ -16,6 +16,11 @@ def new
         accredited_provider_form
       end
 
+      def edit
+        provider
```

Comment:
could we put the provider in a before action?

Reply:
Done: https://github.com/DFE-Digital/publish-teacher-training/pull/3573/commits/e66949a98a36464c026d1c71d7bc81298f6d381b